_datascience/wdl_execution_engines.md

```diff
@@ -0,0 +1,272 @@
+
+> **New to WDL?** Start with our [WDL Workflow guide](/datascience/wdl_workflows/) to learn about the language itself before diving into execution engines.
+
+## Execution Engine Comparison
+
+| Feature | Cromwell | Sprocket | miniWDL |
+|---------|----------|----------|---------|
+| **Best For** | Shared HPC systems, production pipelines | Local execution, development and testing | Alternative for local testing |
+| **Ease of Setup** | Complex (server mode recommended) | Simple (single binary) | Simple (single command install) |
+| **Key Features** | Call caching, server mode, extensive backend support | Lightweight, modern architecture, active development | Fast local execution, good error messages |
+| **Fred Hutch Support** | Full (PROOF, dedicated configs) | Community support | Community support |
+| **Learning Curve** | Steeper | Gentle | Gentle |
+| **When to Use** | Running on Gizmo, AWS Batch; production workflows | Local testing and development; actively maintained | Local testing when Sprocket isn't available |
+
+All three engines follow the [WDL specification](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md), so workflows written for one should work on others with minimal changes.
+
+## Cromwell
+
+[Cromwell](https://cromwell.readthedocs.io/) is a mature workflow execution engine developed by the Broad Institute. It powers platforms like [Terra](https://terra.bio/) and is the most feature-rich option for production environments.
+
+### Why Choose Cromwell?
+
+**Advantages:**
+- **Call caching**: Reuses results from previous identical tasks, saving compute time
+- **Server mode**: Manages multiple workflows simultaneously
+- **Robust backend support**: SLURM, AWS Batch, Google Cloud, Azure, local execution
+- **Production-tested**: Used by major research institutions worldwide
+- **API-driven**: Programmatic workflow submission and monitoring
+
+**Considerations:**
+- More complex setup than miniWDL or Sprocket
+- Requires database configuration for full features
+- Better suited for shared HPC systems than individual workstations
+
+### Cromwell at Fred Hutch
+
+At Fred Hutch, Cromwell is the primary execution engine for WDL workflows on our HPC infrastructure. We provide custom configurations and tools to make it easier to use.
+
+#### Running Cromwell with PROOF
+
+[PROOF](/datascience/proof/) provides a user-friendly web interface for submitting WDL workflows to our cluster:
+
+1. Connect to the Fred Hutch network
+2. Navigate to [proof.fredhutch.org](https://proof.fredhutch.org)
+3. Log in with your Hutch credentials
+4. Upload your WDL file and inputs JSON
+5. Submit and monitor your workflow through the dashboard
+
+PROOF handles all Cromwell server setup, configuration, and job scheduling automatically.
```

Comment:
You may want to briefly explain somewhere what it means to 'start a server', since 'server' conjures images of a physical computer rack. I would personally benefit and someone who doesn't know what cromwell is may benefit too